src/main/java/com/monkeyrizz/codeforest/model/Problem.java

```diff
@@ -1,48 +1,56 @@
-package com.monkeyrizz.codeforest.dto;
-
-public class Problem {
-    private Long id;
-    private String title;
-    private String description;
-
-    public Problem() {
-    }
-
-    public Problem(String title, String description) {
-        this.description = description;
-        this.title = title;
-    }
-
-    public Long getId() {
-        return id;
-    }
-
-    public void setId(Long id) {
-        this.id = id;
-    }
-
-    public String getTitle() {
-        return title;
-    }
-
-    public void setTitle(String title) {
-        this.title = title;
-    }
-
-    public String getDescription() {
-        return description;
-    }
-
-    public void setDescription(String description) {
-        this.description = description;
-    }
-
-    @Override
-    public String toString() {
-        return "Problem{" +
-                "id=" + id +
-                ", title='" + title + '\'' +
-                ", description='" + description + '\'' +
-                '}';
-    }
-}
+package com.monkeyrizz.codeforest.model;
+
+public class Problem {
+    private Long id;
+    private String title;
+    private String description;
+    private Long authorId;
```

Comment:
Please see if we can eliminate the need for having to hard code so many getters and setters by using Lombok's builder annotation.